extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -195,4 +193,75 @@ export class DeployConfigPage extends DacFxConfigPage {
 		this.databaseLoader.loading = false;
 		return true;
 	}
+
+	/*
+	Function that is used to update the window if upgrade radio button is selected.
+	*/
+	private updateUpgradeRadioButton(): void {
+		this.model.upgradeExisting = true;
+		this.formBuilder.removeFormItem(this.databaseComponent);
+		this.formBuilder.addFormItem(this.databaseDropdownComponent, { horizontal: true, componentWidth: 400 });
+
+		// add deploy plan page and remove and re-add summary page so that it has the correct page number
+		this.instance.wizard.removePage(DeployNewOperationPath.summary);
+		let deployPlanPage = this.instance.pages.get(PageName.deployPlan);
+		let summaryPage = this.instance.pages.get(PageName.summary);
+		this.instance.wizard.addPage(deployPlanPage.wizardPage, DeployOperationPath.deployPlan);
+		this.instance.wizard.addPage(summaryPage.wizardPage, DeployOperationPath.summary);
+	}
+
+	/*
+	Function that is used to update the window if new radio button is selected.
+	*/
+	private updateNewRadioButton(): void {
+		this.model.upgradeExisting = false;
+		this.formBuilder.removeFormItem(this.databaseDropdownComponent);
+		this.formBuilder.addFormItem(this.databaseComponent, { horizontal: true, componentWidth: 400 });
+		this.instance.setDoneButton(Operation.deploy);
+
+		// remove deploy plan page and read summary page so that it has the correct page number
+		this.instance.wizard.removePage(DeployOperationPath.summary);
+		this.instance.wizard.removePage(DeployOperationPath.deployPlan);
+		let summaryPage = this.instance.pages.get(PageName.summary);
+		this.instance.wizard.addPage(summaryPage.wizardPage, DeployNewOperationPath.summary);
+	}
+
+	/*
+	Function to create the radio button flex container on the window.
+	*/
+	private createRadioButtonFlexContainer(upgradeRadioButton: azdata.RadioButtonComponent, newRadioButton: azdata.RadioButtonComponent): azdata.FormComponent {
+		let flexRadioButtonsModel = this.view.modelBuilder.flexContainer()
+			.withLayout({
+				flexFlow: 'row',
+			}).withItems([
+				upgradeRadioButton, newRadioButton]
+			).component();
+
+		return {
+			component: flexRadioButtonsModel,
+			title: loc.targetDatabase
+		};
+	}
+
+	/*
+	Function which calls the onDidClick method for upgrade radio button to update it and set the
+	database name.
+	*/
+	private selectUpgradeRadioButton(upgradeRadioButton: azdata.RadioButtonComponent): void {
+		upgradeRadioButton.onDidClick(() => {
```

Comment:
the onDidClick() handlers only need to be set once. I'd prefer if they get set once in the createRadioButtons() function